ternal_apriljoy/restapi/ternal_apriljoy_backend/insert_opmband.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+function insertOpmBand ($servername, $username, $password, $dbname, $data) {
+    
+    $connect = mysqli_connect($servername, $username, $password, $dbname);
+    if (!$connect) return;
+        
+    foreach ($data as $key => $value) {
+        $data[$key] = mysqli_real_escape_string($connect, $value);
+    }
+    
+    $sql = "INSERT INTO opmband (opmband_name, origin, leadvocalist_name, formation_date, 
```

Comment:
```suggestion
    $sql = "INSERT INTO opmband (opmband_name, origin, leadvocalist_name, formation_date, 
```
$sql = "INSERT INTO opmband (opmband_name, origin, leadvocalist_name, 
     formation_date,  first_album)